avro/src/main/java/com/fasterxml/jackson/dataformat/avro/deser/AvroReaderFactory.java

```diff
@@ -1,10 +1,27 @@
 package com.fasterxml.jackson.dataformat.avro.deser;
 
-import java.util.*;
-
+import com.fasterxml.jackson.dataformat.avro.deser.ScalarDecoder.BooleanDecoder;
```

Comment:
Please do not change import statements. I use wildcards for 5 or more types (and sometimes earlier).